src/Uno.UI/UI/Xaml/Controls/Popup/PlacementPopupPanel.cs

```diff
@@ -158,13 +158,13 @@ protected virtual Rect CalculateFlyoutPlacement(Size desiredSize, Size maxSize)
 					case FlyoutPlacementMode.Full:
 						desiredSize = visibleBounds.Size.AtMost(maxSize);
 						finalPosition = new Point(
-							x: (visibleBounds.Width - desiredSize.Width) / 2.0,
-							y: (visibleBounds.Height - desiredSize.Height) / 2.0);
+							x: visibleBounds.Left + (visibleBounds.Width - desiredSize.Width) / 2.0,
```

Comment:
Looks reasonable, can you add a test that verifies the behaviour please

Reply:
There are existing ui test for this. But, not automated one that validates the flyout coords.
And, I've created a new issue for this PR.

Reply:
Make a unit test in 'Uno.UI.Tests' to verify the correct position